packages/terra-consumer-nav/src/components/quick-links/QuickLinks.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './QuickLink.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: null,
+};
+
+const QuickLinks = ({
+  children,
+  ...customProps
+}) => (
+  <div className={cx('quicklinks-container')} {...customProps}>
```

Comment:
if a className is passed as customProp, the internal className 'quicklinks-container' will be overridden. Normally we put customProps at the begining. So the internal attrs will have higher priority